paprika/amber.py

```diff
@@ -82,6 +83,27 @@ def restraint_file(self):
     def restraint_file(self, value):
         self._restraint_file = value
 
+    @property
```

Comment:
Just curious, is there a way of marking a property as optional?

Reply:
I don't know. Are you thinking of placing a logic for "either use a `restraint_file` or `plumed_file`"

Reply:
That, and I was also just wondering whether there's some elegant way to make a class attribute optional. Not really a big deal but could be useful for autogenerating documentation. I've used this before with `argparse`, for example: https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
I see, yeah that would be a nice feature so you don't need to keep writing this in the docstring for every optional property. Maybe I can raise an exception if the user specifies both `restraint_file` and `plumed_file`. What do you think?

Reply:
What happens if they are both specified now? I assume it will write a simulation `in` file that will point to both the restraint file and the PLUMED file. That's probably going to do something bad. If that's the case, then yeah, we should probably raise the exception, just as you suggested.

Reply:
If both are specified then both files will be written to the `in` file. Okay, I'll raise an exception in the `amber.py` code.

Reply:
Done.